resources/airbnb.swiftformat

```diff
@@ -52,7 +52,7 @@
 --rules wrap
 --rules wrapArguments
 --rules wrapAttributes
---rules wrapMultilineStatementBraces
```

Comment:
The `wrapMultilineStatementBraces` rule was removed in nicklockwood/SwiftFormat#1071 in favor of an option on the `braces` rule (which is enabled by default). So this change preserves the existing behavior for multi-line statements/expressions, and now also applies to single-line statements/expressions.